### DISCUSSION TITLE
Unexpected EOF on client connection with an open transaction

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#931] [Frontier TestNet - Bug Reporting]
Not sure if this is a bug/problem or not but I'm regularly seeing messages like this in my log output:

`coordinator_db       | 2022-04-27 18:56:11.394 UTC [29943] LOG:  unexpected EOF on client connection with an open transaction`

**Comment by mzxyz** [comment]
This error is from db is allright
